stacker/tests/lookups/test_registry.py

```diff
@@ -1,12 +1,35 @@
 import unittest
 
-from stacker.lookups.registry import LOOKUP_HANDLERS
+from mock import patch, MagicMock
+
+from stacker.exceptions import (
+    UnknownLookupType,
+    FailedVariableLookup,
+)
+
+from stacker.lookups.registry import (
+    LOOKUP_HANDLERS,
+    resolve_lookups,
+)
+
+from stacker.variables import Variable
+
+from ..factories import (
+    mock_context,
+    mock_provider,
+)
 
 
 class TestRegistry(unittest.TestCase):
+    def setUp(self):
+        self.ctx = mock_context()
+        self.provider = mock_provider()
 
     def test_autoloaded_lookup_handlers(self):
-        handlers = ["output", "xref"]
+        handlers = [
```

Comment:
I realized we weren't testing all the auto-loaded lookups, so I went ahead and added them.